psd-web/app/assets/javascripts/cookie_banner.js

```diff
@@ -10,6 +10,7 @@ function cookieBanner () {
     document.cookie = `seen_cookie_message = true;${expires};path=/`
   }
   function hideCookieBanner () {
```

Comment:
Not sure why this isn't required, but I think it'd be good practice, to make the `event` reference more explicit:
```suggestion
  function hideCookieBanner (event) {
```